components/ibm-components/spark/store_spark_model/component.yaml

```diff
@@ -0,0 +1,45 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: 'Store Spark Model - Watson Machine Learning'
```

Comment:
Would it make more sense in WML than Spark?

Reply:
Sure, Since we don't have a WML directory, should I put this component repository one level above? Or I should create a WML directory and group the existing WML components together?

Reply:
Well, WML is the Watson components. There is a store in it, but seems it won`t work with Spark store as is, and we need another one specific to Spark?

Reply:
The reason I didn't implement it with the WML store component because storing custom spark model requires us to load the model and training data into memory. It means we will need to have a full Spark and pyspark installed in our base image which is not needed for general storing use case.

For more details on storing custom spark model, you can visit
https://dataplatform.cloud.ibm.com/docs/content/wsj/analyze-data/ml-import-spark-mllib.html

Reply:
@animeshsingh So which directory you want to put with this component?

Reply:
Its fine under Spark